centraldashboard/rockcraft.yaml

```diff
@@ -1,33 +1,40 @@
+# Dockerfile https://github.com/kubeflow/kubeflow/blob/v1.7-branch/components/centraldashboard/Dockerfile
 name: kubeflow-central-dashboard
 summary: Kubeflow Landing Page
 description: |
   This component serves as the landing page and central dashboard for Kubeflow deployments.
   It provides a jump-off point to all other facets of the platform.
-version: 1.7.0_22.04_1 # version format: <KF-upstream-version>_<base-version>_<Charmed-KF-version>
+version: "1.7"
 license: Apache-2.0
-base: ubuntu:22.04
+base: ubuntu@22.04
+platforms:
+  amd64:
+run-user: _daemon_
+
 services:
   serve:
-    override: replace
-    summary: "centraldashboard service"
+    override: merge
```

Comment:
Is there a reason for changing this?

Reply:
I don't think we follow a standard, and since I was following another example I had, I left it like that. We should probably discuss how we want to approach the `override` strategy, until then, should I keep `replace`? Wdyt?

Reply:
Makes sense. I added a [note](https://docs.google.com/document/d/1Zm11x2sRxue3sUEWe776-6ZHPwyDl6hVXP41z2zhCno/edit#heading=h.6nwxcci67ohi) for the next how do you rock meeting, so we can discuss it as a team.